examples/power.py

```diff
@@ -0,0 +1,25 @@
+import os, sys, inspect, logging, time
+
+lib_folder = os.path.join(os.path.split(inspect.getfile( inspect.currentframe() ))[0], '..')
+lib_load = os.path.realpath(os.path.abspath(lib_folder))
+
+if lib_load not in sys.path:
+    sys.path.insert(0, lib_load)
+
+FORMAT = '%(levelname)-8s %(message)s'
+logging.basicConfig(format=FORMAT)
+logger = logging.getLogger('USB Hub Status')
+logger.setLevel(logging.DEBUG)
+
+import capablerobot_usbhub, power
```

Comment:
Importing `power` isn't needed, and in my environment causes the for loop below to run twice (as the example file itself is imported and run from within the example file).